### PR TITLE
Script to load locally hacked eBPF Programs

### PR DESCRIPTION
**Pull request by vishnoianil**
Script to load locally hacked eBPF Programs to the merbridge pods that is already deployed through daemonset.
This script just a tool to save some time for devs by avoiding redeploying the merbridge daemonset (specially in the non-Kind setup) or skipping the manual steps to load the individual programs on each pod.

Signed-off-by: Anil Kumar Vishnoi <vishnoianil@gmail.com>

### CODE REVIEW

File: hack/reload-prog.sh [hunk 1]
```diff
@@ -0,0 +1,157 @@
+#/bin/sh
```
Comment (kebe7jun) [inline hack/reload-prog.sh:1]: missing license header.

File: hack/reload-prog.sh [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+then
+	echo "Please ensure that kubectl/clang/bpftool are installed and in the path."
+	exit
+fi
+
+declare -a podlist
+
+get_pod_list() {
+        if [ "$1" = "all" ] ;then
+                podlist=( `kubectl get pods -n istio-system | grep merbridge | awk '{print $1}'` )
```
Comment (kebe7jun) [inline hack/reload-prog.sh:77]: use `-l app=merbridge` instead?

File: hack/reload-prog.sh (hunk 1 above)
Comment (vishnoianil) [reply hack/reload-prog.sh:14]: Fixed

**Review by kebe7jun — APPROVED** [review]

**Comment by kebe7jun** [comment]
Thanks!

**Pull request merged**